src/main/resources/fabric.mod.json

```diff
@@ -20,8 +27,8 @@
     "autorelog.mixins.json"
   ],
   "depends": {
-    "fabricloader": ">=${loader_version}",
-    "fabric": "*",
+    "fabricloader": ">=0.12.23",
```

Comment:
This version should move to gradle.properties

Reply:
Understandable.